docs/UserGuide.md

```diff
@@ -880,18 +930,18 @@ Some commands require you to include parameters. These parameters are identified
+|--------|-----------------------------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| n\|    | Name                        | 80          | Should only contain alphanumeric characters and spaces.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| n\|    | Exam Name                   | 30          | Should only contain alphanumeric characters and spaces.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| p\|    | Phone Number                | 30          | It can start with an optional `+` to indicate the country code, with the rest only containing numbers. It should be at least 3 digits long.                                                                                                                                                                                                                                                                                                                                                                                      |
+| e\|    | Email                       | 50 before and after `@` | **Format:** local-part@domain<br/> **Constraints for local part:**<br/> • Should only contain alphanumeric characters, and the characters `+`, `_`, `.` and `-`<br/> • Should not start with special characters<br/> **Constraints for domain:**<br/> • Made up of domain labels followed by periods<br/> • Must end with a domain label of at least 2 characters long<br/> • Should start and end with alphanumeric characters<br/> • Domain label should consist of alphanumeric characters separated only by hyphens, if any |
```

Comment:
Could the max length for email be updated to 100 in total? I have changed the code such that it minimizes confusion. Thanks!